src/app/js/formats/distributionChart/global-radar-chart/RadarChartAdmin.js

```diff
@@ -75,10 +76,86 @@ class RadarChartAdmin extends Component {
         updateAdminArgs('scale', scale, this.props);
     };
 
+    setColors = (_, colors) => {
+        updateAdminArgs('colors', colors, this.props);
+    };
+
+    setColorsWithColorPicker = () => {
+        updateAdminArgs('colors', this.getColorsFromPicker(), this.props);
+    };
+
+    getColorsFromPicker() {
+        let numberOfPickers = this.getColorsArray().length;
+        let res = '';
+        for (var i = 0; i < numberOfPickers; i++) {
+            if (i == 0) {
+                res += this.state.colors[i].color;
+            } else {
+                res += ' ' + this.state.colors[i].color;
+            }
+        }
+        this.setColors;
+        return res;
+    }
+
+    getColorsArray() {
+        return this.props.args.colors.split(' ');
+    }
+
+    constructor(props) {
+        super(props);
+        isFirstLoading = true; // to initialize values on admin UI opening
+        const colorsArray = this.getColorsArray();
+
+        this.state = {
+            colors: [{ color: colorsArray[0] }],
+        };
```

Comment:
Why make a case here for the first color, and a second for the remaing colors?

Reply:
(See upper issue)